pkg/nimble/contrib/nimble_riot.c

```diff
@@ -119,4 +126,13 @@ void nimble_riot_init(void)
 #ifdef MODULE_NIMBLE_SVC_IPSS
     ble_svc_ipss_init();
 #endif
+
+#ifdef MODULE_NIMBLE_AUTOCONN_AUTOSTART
```

Comment:
Should this be defined by default if `autoconn` is used?

Reply:
Yes, I guess that would be a viable thing to do. I think I will change the semantics so its actually 
```c
#ifndef MODULE_NIMBLE_AUTOCONN_NOAUTOSTART
...
```
As the not using it is probably the more exotic use-case, right?!

Reply:
Yep!